content/06.methods.md

```diff
@@ -113,6 +113,12 @@ Note that GATK v4.0.12.0 was used for all of these steps except VariantFiltratio
 This single-sample workflow is available in the [D3b GitHub repository](https://github.com/d3b-center/OpenPBTA-workflows).
 References can be obtained from the [Broad Genome References on AWS](https://s3.console.aws.amazon.com/s3/buckets/broad-references/hg38/v0/) bucket, and a general description of references can be found at https://s3.amazonaws.com/broad-references/broad-references-readme.html.
 
+##### Identification of pathogenic germline variants in hypermutant samples
+
+We functionally annotated germline variants with SnpEff [@pmid:22728672], followed by a pathogenic/likely pathogenic (P/LP) annotation with ANNOVAR [@doi:10.1093/nar/gkq603] and InterVar [@doi:10.1016/j.ajhg.2017.01.004].
+To identify pathogenic variants in hypermutant samples, we removed variants with a minor allele frequency of >= 0.1% in ExAC [@doi:10.1093/nar/gkw971] and any gnomAD population [@doi:10.1038/s41586-020-2308-7], as those are unlikely to be pathogenic.
+If P/LP calls conflicted betweeen ANNOVAR and InterVar, we manually reviewed them, integrating information in ClinVar [@doi:10.1093/nar/gkx1153].
```

Comment:
I am still a bit confused here, so I think we may need more detail. My understanding of InterVar is that it uses ANNOVAR as input, so I am not clear on what would constitute a conflict: which ANNOVAR annotation is being compared to the InterVar call? 

I assume here that P/LP calls are all retained as potentially pathogenic, but it is somewhat ambiguous here.

My suggested structure is below, but please update it with the process that was used:
```
We annotated germline variants with SnpEff, followed by annotation with ANNOVAR and InterVar.
We selected potentially pathogenic variants in hypermutant samples as those annotated as XXXXX by ANNOVAR or XXXXX by InterVar
As variants present in the population are unlikely to pathogenic, we removed variants with minor allele freq...
In the case of conflicting annotations between ANNOVAR  and InterVar, we manually reviewed...
```